### DISCUSSION TITLE
package vulnerability

### DISCUSSION THREAD

**Discussion by jwugit** [vercel/next.js#17124] [Help]
anyone seeing this recently ?
using  "next": "^9.5.3"

![image](https://user-images.githubusercontent.com/28541026/93255145-78498e00-f767-11ea-816b-1edc4d669c7a.png)

**Comment by dendi** [comment]
same here. 

![image](https://user-images.githubusercontent.com/4526612/93731741-89c3e780-fbf8-11ea-837d-097c8083aaba.png)

is this still unfixed?
